contrib/ros/envs/ros_env.py

```diff
@@ -0,0 +1,57 @@
+import numpy as np
+import rospy
+
+# Every environment class in rllab should inherit Serializable and call Serializable.quick_init(self, locals())
+from rllab.core.serializable import Serializable
+from rllab.envs.base import Env
+from rllab.misc.ext import get_seed
+
+
+class RosEnv(Env, Serializable):
+    """
+    Superclass for all ros environment
+    """
+
+    def __init__(self):
+        Serializable.quick_init(self, locals())
+
+        self.np_random_generator = np.random.RandomState(get_seed())
```

Comment:
Why do you store a random generator? numpy's seed is globally-initialized by rllab.